aws/rust-runtime/clippy.toml

```diff
@@ -0,0 +1,7 @@
+disallowed-methods = [
```

Comment:
Nice!

Reply:
Should there be one of these in `rust-runtime/` too?

Reply:
refactored to have a single file & symlinks